### PR TITLE
Modify get_overview to support duplicate names and provide a simplify…

### PR DESCRIPTION
**Pull request by lb-pno**
This is a fix for the new `get_overview()` method.
It now supports the use of duplicate names for reviews and uses "total" instead of "all" following feedback from QA ([https://labelbox.atlassian.net/browse/PLT-1012](https://labelbox.atlassian.net/browse/PLT-1012)).

There is now an optional parameter to only display the details of the queues when needed: 

```python

# Default behaviour
overview = project.get_overview()
print("Output type:", type(overview).__name__)
print(overview.dict())
```
Result:
```bash
Output type: ProjectOverview
{'to_label': 1619, 'in_review': 1757, 'in_rework': 162, 'skipped': 0, 'done': 45844, 'issues': 0, 'labeled': 47764, 'total_data_rows': 49383}
```

With the details:
In this mode, "in_review" and "in_rework" are a dictionary with the list of the queues (`data`) with the associated number of data rows, and the `total` number of data rows.

```python
overview = project.get_overview(details=True)
print("Output type:", type(overview).__name__)
print(overview.dict())
```
Result:
```bash
Output type: ProjectOverviewDetailed
{'to_label': 1619, 'in_review': {'data': [{'April review - Douglas': 0}, {'April review - Joyce': 13}, {'May reviews - Daisy': 0}, {'April review - Sam': 22}, {'April review - Wafula': 12}, {'April/May review - Lilian': 556}, {'April review - kelvin': 0}, {'April/May review - Charles': 0}, {'April review - Samuel': 3}, {'April/May review - Wakairu': 1151}], 'total': 1757}, 'in_rework': {'data': [{'Rework (all rejected)': 162}], 'total': 162}, 'skipped': 0, 'done': 45844, 'issues': 0, 'labeled': 47764, 'total_data_rows': 49383}
```

### CODE REVIEW

File: libs/labelbox/src/labelbox/schema/project.py [hunk 1]
```diff
@@ -1753,62 +1753,97 @@ def __check_data_rows_have_been_processed(
         return response["queryAllDataRowsHaveBeenProcessed"][
             "allDataRowsHaveBeenProcessed"]
 
-    def get_overview(self) -> ProjectOverview:
-        """ Return the number of data rows per task queue, and issues of a project
-            Equivalent of the Overview tab of a project
+    def get_overview(self, details=False) -> Union[ProjectOverview, ProjectOverviewDetailed]:
+            """Return the overview of a project.
```
Comment (adrian-chang) [inline libs/labelbox/src/labelbox/schema/project.py]: Formatting seems wrong here

File: libs/labelbox/src/labelbox/schema/project.py [hunk 2]
```diff
@@ -1753,62 +1753,97 @@ def __check_data_rows_have_been_processed(
...
+            overview["to_label"] = overview.pop("unlabeled")
+            overview["total_data_rows"] = overview.pop("all")        
 
-        return ProjectOverview(**overview)
+            if not details:
+                return ProjectOverview(**overview)
+            else:
+                # Build queue details for review and rework queues
+                for category in ["rework", "review"]:
+                    overview[f"in_{category}"] = _build_queue_details(f"in_{category}",
```
Comment (adrian-chang) [inline libs/labelbox/src/labelbox/schema/project.py]: is a nested function needed given how simple it is?

**Review by adrian-chang — CHANGES_REQUESTED** [review]

File: libs/labelbox/src/labelbox/schema/project.py (hunk 1 above)
Comment (lb-pno) [reply libs/labelbox/src/labelbox/schema/project.py]: Fixed

File: libs/labelbox/src/labelbox/schema/project.py (hunk 2 above)
Comment (lb-pno) [reply libs/labelbox/src/labelbox/schema/project.py]: Fixed

**Review by adrian-chang — APPROVED** [review]
Great work @paulnoirel. Let's merge in once tests are green (I reran).

**Pull request merged**